openshift_tools/monitoring/ocutil.py

```diff
@@ -69,6 +71,7 @@ def _run_cmd(self, cmd, base_cmd='oc', ):
         if self.logger:
             self.logger.debug("ocutil._run_cmd( {} )".format(cmd))
 
+        # FIXME: pylint redefined-variable-type doesn't like this, but it's working?
```

Comment:
we shouldn't need to disable this pylint. appeasing pylint should be as easy as changing the variable name to 'cmd_split' and changing the two references right below the assignment:

```
cmd_split = shlex.split(cmd)

if self.verbose:
  print "Running command: {}".format(str(cmd_split))

try:
  return subprocess.check_output(cmd_split)
```